pyrefinebio/token.py

```diff
@@ -14,56 +15,81 @@ class Token:
 
     Please review refine.bio's [Terms of Use](https://www.refine.bio/terms) and [Privacy Policy](https://www.refine.bio/privacy)
     before use of these tokens.
-    """
 
-    @classmethod
-    def create_token(cls, email_address):
-        """creates a token and emails the terms and conditions to a specified email.
+    Create a new Token
 
-        parameters:
+        ex:
+        >>> import pyrefinebio
+        >>> token = pyrefinebio.Token(email_address="foo@bar.com")
 
-            email_address (str): the email that the terms and conditions should be sent to.
-        """
+    Create a Token and activate it
 
-        response = post_by_endpoint("token").json()
+        ex:
+        >>> import pyrefinebio
+        >>> token = pyrefinebio.Token(email_address="foo@bar.com")
+        >>> token.agree_to_terms_and_conditions()
 
-        token_id = response["id"]
+    Save a Token to the config file
 
-        if email_address:
-            terms = response["terms_and_conditions"]
-            # email terms
+        ex:
+        >>> import pyrefinebio
+        >>> token = pyrefinebio.Token(email_address="foo@bar.com")
+        >>> token.save_token()
 
-        return token_id
+    Load the Token that is currently saved to the config file
 
-    @classmethod
-    def agree_to_terms_and_conditions(cls, api_token):
+        ex:
+        >>> import pyrefinebio
+        >>> token = pyrefinebio.Token.load_token()
+    """
+
+    def __init__(
+        self,
+        email_address=None,
+        id=None,
+    ):
+        if id:
+            try:
+                response = get_by_endpoint("token/" + str(id))
+            except NotFound:
+                raise BadRequest(
+                    "Token with id '" + str(id) + "' does not exist in RefineBio. " 
+                    "Please create a new token."
+                )
+        else:
+            response = post_by_endpoint(
+                "token",
+                payload={
+                    "email_address": email_address
+                }
+            ).json()
+
+        self.email_address = email_address
+
+        self.token_id = response["id"]
```

Comment:
I think this should just be `self.id`. Otherwise its `token.token_id` instead of just `token.id`.